### DISCUSSION TITLE
Voibos Höhenabfrage: Differenz zwischen Oberflächen- und Geländemodell rechnen

### DISCUSSION THREAD

**Discussion by fanie-mak** [e-netze/webgis-community#161] [Q&A]
Hallo, 
wir verwenden den Höhenservice von VOIBOS für die Höhenabfrage (h.xml): 
```
<heightabovedatum type="voibos" srs="31256" server="https://voibos.rechenraum.com/voibos/voibos" service="hoehenservice" rastertheme="hoeheDTM" expression="{0:0.00}, ([flugjahr])" name="Gelände (müA, Jahr)"/>
	
  <heightabovedatum type="voibos" srs="31256" server="https://voibos.rechenraum.com/voibos/voibos" service="hoehenservice" rastertheme="hoeheDSM" expression="{0:0.00}, ([flugjahr])" name="Oberfläche (müA, Jahr)"/>
```

Wäre es möglich eine Dritte Zeile einzufügen, wo WebGIS die Differenz zwischen beiden Werte berechnet?

Danke und LG

**Comment by gis-eni** [comment]
Nein, das ist nicht vorgesehen. Die Abfragen laufen im Hintergrund unabhängig voneinander ab und die Ergebnisse werde angezeigt.
Wenn du eine Differenz haben willst, muss das der Dienst hergeben.

**Reply by fanie-mak** [reply]
Ok danke für die Info...
